x-pack/plugins/observability_solution/observability/public/components/custom_threshold/components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -109,10 +112,13 @@ export default function AlertDetailsAppSection({
     color: chroma(transparentize('#F04E981A', 0.2)).hex().toUpperCase(),
     id: `custom_threshold_${alertEnd ? 'recovered' : 'active'}_alert_range_annotation`,
   };
-
   const annotations: EventAnnotationConfig[] = [];
   annotations.push(alertStartAnnotation, alertRangeAnnotation);
 
+  useEffect(() => {
+    setRelatedAlertsKuery(getRelatedAlertKuery(tags, groups));
+  }, [groups, setRelatedAlertsKuery, tags]);
```

Comment:
Do we need to check for empty groups or tags array? Or that's not possible?

Reply:
This will be handled in getRelatedAlertKuery:

```
getRelatedAlertKuery = (tags?: string[], groups?: Group[])
```